apps/sample-ui/src/mocks/resolver.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  DefaultRequestBody,
+  PathParams,
+  ResponseComposition,
+  RestContext,
+  RestRequest,
+} from 'msw'
+
+export interface IRecordedResponse {
+  name: string
+  originalResponse: any
+}
+
+function parseRequestName(req: RestRequest<DefaultRequestBody, PathParams>) {
+  const path = req.url.pathname
+    .substring(1, req.url.pathname.length)
```

Comment:
```suggestion
    .substring(1, req.url.pathname.length) // remove leading slash
```

Reply:
Just to remind me what this does in future ^^

Reply:
Updated! This function is supposed to create a unique request name based on msw RestRequest object to identify each REST API request so that we can manage the original request response in LS like below. 
<img width="831" alt="image" src="https://user-images.githubusercontent.com/6326478/154940569-1b73ae27-a1bd-4e90-a607-9ad7b1907eca.png">

This is still a just initial implementation and it does not cover query params and wildcard URL path segment etc yet but I'll add support for those later :)

Reply:
After looking at your comment https://github.com/kiseki-fans/bleublanc/pull/13#discussion_r810999421, I think we won't need this fn for blueblanc cypress plugin, it should be able to refer mock file path from cy.intercept and map it to actual API request